community-managed-token/program/src/lib.rs

```diff
@@ -259,3 +269,31 @@ pub fn process_revoke(accounts: &[AccountInfo]) -> ProgramResult {
     revoke(token_account, owner, token_program)?;
     freeze(freeze_authority, mint, token_account, token_program, &seeds)
 }
+
+pub fn process_migrate_authority(accounts: &[AccountInfo]) -> ProgramResult {
```

Comment:
just to confirm, this is a one way migration as the authority will essentially leave the CMT program?

Reply:
yes, outbound migration. Inbound doesn't need an entrypoint, can do it directly with spl-token program